opennms-webapp/src/main/java/org/opennms/web/filter/FilterUtil.java

```diff
@@ -50,4 +51,14 @@ public static String toFilterURL(List<String> filters) {
         }
         return( buffer.toString() );
     }
+
+    public static String[] parse(String filterString) {
```

Comment:
Please add a test case where `filterString` can have multiple filter expressions

Reply:
Converted test cases to use multiple filters